common-npm-packages/artifacts-common/credentialProviderUtils.ts

```diff
@@ -119,36 +119,54 @@ export async function configureCredProvider(protocol: ProtocolType, serviceConne
 }
 
 /**
- * Configure the credential provider to provide credentials for feeds within the pipeline's organization,
- * as well as for a single Entra backed service connection.
+ * Configure credentials for the given feed using the provided 'Azure Devops' service connection.
+ * Only external feeds are supported, will throw if the feed provided is internal.
  */
-export async function configureEntraCredProvider(protocol: ProtocolType, feedUrl: string, entraWifServiceConnectionName: string) {
-    await configureCredProviderForSameOrganizationFeeds(protocol);
+export async function configureEntraCredProvider(protocol: ProtocolType, entraWifServiceConnectionName: string, feedUrl: string | undefined) : Promise<boolean> {
```

Comment:
are we certain that we are the only ones using this function and that changing this contract ordering will not break others ?

Reply:
Yep, I created this function last time around and just confirmed we're the only ones using this